app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailEvent.kt

```diff
@@ -1,9 +1,17 @@
 package com.example.tasky.agenda.presentation.itemdetail.viewmodel
 
 import com.example.tasky.agenda.presentation.itemdetail.model.VisitorType
+import java.time.LocalDate
+import java.time.LocalTime
 
 sealed class EventDetailEvent {
     object OnCloseClick : EventDetailEvent()
     object CloseClickResolved : EventDetailEvent()
+    data class ShowTimePicker(val isFrom: Boolean) : EventDetailEvent()
+    object HideTimePicker : EventDetailEvent()
+    data class TimeSelected(val time: LocalTime) : EventDetailEvent()
+    data class ShowDatePicker(val isFrom: Boolean) : EventDetailEvent()
```

Comment:
The isFrom is hard to understand

Yes, you can I can understand that it´s either "from" or "to". But because we know the code.

If I was a teammate, I would never understand that

So here you have 2 possible solutions:

1- Have 2 events: ShowFromDatePicker and ShowToDatePicker
2- Have a sealed class that represents the "To" and "From" values. So something like:
```
sealed class DateSelector {
    object From: DateSelector()
    object To: DateSelector()
}
```
And then have the ShowDatePicker receive this sealed class: 
```
data class ShowDatePicker(val selector: DateSelector) : EventDetailEvent()
```

And then you'd check if selector is from, do something, if it's to, do another thing



To be honest, to make it simpler, I'd just have 2 different Events